helpers/filterSeats.js

```diff
@@ -15,11 +14,11 @@ module.exports = function( dust ) {
   * @example {@_filterSeats packageRates=packageRatesReply.packageRates venueProducts=venueProductsReply.venueProducts roomRates=packageRatesReply.linked.roomRates ticketRates=packageRatesReply.linked.ticketRates} {/_filterSeats} output loop of sections with seats inside sorted by price
   */
 
-  dust.helpers._filterSeats = function( chunk, context, bodies, params ) {
+  dust.helpers._filterSeats = function(chunk, context, bodies, params) {
     params = params || {};
 
     var packageRates = params.packageRates;
-    var venueProduct = _.first( params.venueProducts );
+    var venueProduct = _.head(params.venueProducts);
```

Comment:
I've had a think over the weekend on this. We are about to start the process to dump dust for handlebars. The test that you have suggested isn't really testing the intention of the code, and due to time constraints I won't be able to add a proper set of unit tests for these helpers.

I can see it is working by npm linking the-wall-tempater-dustjs and will make sure we properly QA the sites that the Wall serves.

When we need similar functionality in Handlebars we will make every helper is properly unit tested.